assets/check

```diff
@@ -35,4 +36,37 @@ fi
 
 echo "Using the API endpoint: ${endpoint}/api/instana/version"
 
-curl --silent --show-error --fail --header "Authorization: apiToken ${api_token}" "${endpoint}/api/instana/version" | jq -e '.branch' | sed s/release-// | jq '[{ release: . }]' >&3
+readonly data=$(curl --silent --show-error --fail --header "Authorization: apiToken ${api_token}" "${endpoint}/api/instana/version")
+readonly branch=$(echo "${data}" | jq -er '.branch' | sed s/release-//)
+
+readonly imageTag=$(echo "${data}" | jq -er '.imageTag')
+readonly versions=( ${imageTag//./ } )
+
+readonly major="${versions[0]}"
+readonly minor="${versions[1]}"
+readonly patch="${versions[2]}"
+
+version=""
+case "$granularity" in
```

Comment:
Please `${granularity}`; I know it is the same semantics, but this notation makes me a sad panda